src/account.ts

```diff
@@ -106,6 +106,21 @@ export abstract class Account {
         return this.starknetContract.address;
     }
 
+    async getAccountBalance(): Promise<StringMap> {
+        const abiPath = process.cwd() + "/contract-artifacts/ERC20/ERC20_abi.json";
```

Comment:
Why not use `hre.starknet.getContractFactory`?

Reply:
There's no particular good reason. This has been refactored👍